todo-server/auth-service/src/main/kotlin/io/supersimple/auth/attribute/AuthenticationKey.kt

```diff
@@ -1,5 +1,6 @@
 package io.supersimple.auth.attribute
 
-import io.ktor.util.AttributeKey
-
-val AuthenticationKey = AttributeKey<String>("jwt-auth")
+/**
+ * Name of the default JWT authentication provider
+ */
+const val DefaultAuthProvider = "default-jwt-auth"
```

Comment:
Yeah if we were actually using dependency injection via attributes (like we do with the `NotesRepositoryKey`) I think it would make sense to do it this way. But you're probably right - better to use null / delete it altogether since it's so unlikely that we'll use more than one auth provider anyway.